### PR TITLE
Some Changes

### PR DESCRIPTION
**Pull request by UdonCodes**
**Added**
+ [jellyfin-plugin-media-cleaner](https://github.com/shemanaev/jellyfin-plugin-media-cleaner)
+ [jellyfin-plugin-onepace](https://github.com/jwueller/jellyfin-plugin-onepace)
+ [JellyRoller](https://github.com/LSchallot/JellyRoller)
+ [ytdlp2STRM](https://github.com/fe80Grau/ytdlp2STRM)
+ [Jellyman](https://github.com/Smiley-McSmiles/jellyman)

**Removed**
+ BlueBerry Theme (because it is archived now)

**Other**
+ Updated Outdated Indicators

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -31,6 +31,8 @@
 - [Jellyfin-Newsletter-Plugin ` 🔸 ` `📅`](https://github.com/Cloud9Developer/Jellyfin-Newsletter-Plugin) - Newsletters for recently added media.
 - [jellyfin-plugin-enigma2 `📅`](https://github.com/knackebrot/jellyfin-plugin-enigma2) - Supports Vu+ & Enigma2 live TV streamers.
 - [jellyfin-plugin-ldapauth](https://github.com/jellyfin/jellyfin-plugin-ldapauth) - Allows the use of ldap as an auth provider.
+- [jellyfin-plugin-media-cleaner]() - Automatically removes played Media after time period.
```
Comment (darmiel) [inline README.md:34]: This entry is missing the URL

Comment (UdonCodes) [reply README.md]: Fixed it, sry was kinda tired that day ,:D

File: README.md [hunk 2]
```diff
@@ -31,6 +31,8 @@
 - [Jellyfin-Newsletter-Plugin ` 🔸 ` `📅`](https://github.com/Cloud9Developer/Jellyfin-Newsletter-Plugin) - Newsletters for recently added media.
 - [jellyfin-plugin-enigma2 `📅`](https://github.com/knackebrot/jellyfin-plugin-enigma2) - Supports Vu+ & Enigma2 live TV streamers.
 - [jellyfin-plugin-ldapauth](https://github.com/jellyfin/jellyfin-plugin-ldapauth) - Allows the use of ldap as an auth provider.
+- [jellyfin-plugin-media-cleaner](https://github.com/shemanaev/jellyfin-plugin-media-cleaner) - Automatically removes played Media after time period.
+- [jellyfin-plugin-onepace](https://github.com/jwueller/jellyfin-plugin-onepace) - Plugin for the [One Pace](https://onepace.net) Projekt
```
Comment (darmiel) [inline README.md:35]:
```suggestion
- [jellyfin-plugin-onepace](https://github.com/jwueller/jellyfin-plugin-onepace) - Plugin for the [One Pace](https://onepace.net) Project.
```

File: README.md [hunk 3]
```diff
@@ -68,6 +72,7 @@
 - [TitleCardMaker](https://github.com/CollinHeist/TitleCardMaker) - Automated title card maker for Plex, Jellyfin, and Emby.
 - [wizarr](https://github.com/Wizarrrr/wizarr) - Advanced user invitation and management system.
 - [ytdl-sub](https://github.com/jmbannon/ytdl-sub) - Automate downloading and metadata generation with YoutubeDL.
+- [ytdlp2STRM](https://github.com/fe80Grau/ytdlp2STRM) - streams YT, Twitch and more via ytdlp to Jellyfin.
```
Comment (darmiel) [inline README.md:75]:
```suggestion
- [ytdlp2STRM](https://github.com/fe80Grau/ytdlp2STRM) - Streams YouTube, Twitch and more via yt-dlp to Jellyfin.
```

**Review by darmiel — CHANGES_REQUESTED** [review]
just a few nits

**Review by darmiel — APPROVED** [review]

**Pull request merged**